docs/src/tutorials/conic/dualization.jl

```diff
@@ -0,0 +1,257 @@
+# two scalar equality constraints, and a constraint that `X` is positive
+# semidefinite.
+
+# The dual of `model_primal` is:
+
+model_dual = Model()
+@variable(model_dual, y[1:2])
+@objective(model_dual, Min, sum(y))
+@constraint(model_dual, dual_c, [y[1]-1 1; 1 y[2]-1] in PSDCone())
+print(model_dual)
+
+# This problem has two scalar decision variables, and a 2x2 positive
+# semidefinite matrix constraint.
+
+# !!! tip
+#     If you haven't seen conic duality before, try deriving the dual problem
+#     based on the description in [Duality](@ref). You'll need to know that the
+#     dual cone of [`PSDCone`](@ref) is the [`PSDCone`](@ref).
+
+# When we solve `model_primal` with `SCS.Optimizer`, SCS reports three variables
+# (`variables n: 3`), five rows in the constraint matrix (`constraints m: 5`),
+# and five non-zeros in the matrix (`nnz(A): 5`):
+
+set_optimizer(model_primal, SCS.Optimizer)
+optimize!(model_primal)
+
+# (There are five rows in the constraint matrix because SCS expects problems in
+# geometric conic form, and so JuMP has reformulated the `X, PSD` variable
+# constraint into the affine constraint `X .+ 0 in PSDCone()`.)
+
+# The solution we obtain is:
+
+value.(X)
+
+#-
+
+dual.(primal_c)
+
+#-
+
+objective_value(model_primal)
+
+# When we solve `model_dual` with `SCS.Optimizer`, SCS reports two variables
+# (`variables n: 2`), three rows in the constraint matrix (`constraints m: 3`),
+# and two non-zeros in the matrix (`nnz(A): 2`):
+
+set_optimizer(model_dual, SCS.Optimizer)
+optimize!(model_dual)
+
+# and the solution we obtain is:
+
+dual.(dual_c)
+
+#-
+
+value.(y)
+
+#-
+
+objective_value(model_dual)
+
+# This particular problem is small enough that it isn't meaningful to compare
+# the solve times, but in general, we should expect `model_dual` to solve faster
+# than `model_primal` because it contains fewer variables and constraints. The
+# difference is particularly noticeable on large-scale optimization problems.
+
+# ## `dual_optimizer`
+
+# Manually deriving the conic dual is difficult and error-prone. The package
+# [Dualization.jl](@ref) provides the `Dualization.dual_optimizer` meta-solver,
+# which wraps any MathOptInterface-compatible solver in an interface that
+# automatically formulates and solves the dual of an input problem.
+
+# To demonstrate, we use `Dualization.dual_optimizer` to solve `model_primal`:
+
+set_optimizer(model_primal, Dualization.dual_optimizer(SCS.Optimizer))
+optimize!(model_primal)
+
+# The performance is the same as if we solved `model_dual`, and the correct
+# solution is returned to `X`:
+
+value.(X)
+
+#-
+
+dual.(primal_c)
+
+# Moreover, if we use `dual_optimizer` on `model_dual`, then we get the same
+# performance as if we had solved `model_primal`:
+
+set_optimizer(model_dual, Dualization.dual_optimizer(SCS.Optimizer))
+optimize!(model_dual)
+
+#-
+
+dual.(dual_c)
+
+#-
+
+value.(y)
+
+# ## A mixed example
```

Comment:
I added this additional section with a non-obvious example

Reply:
I like it!